packages/random/package.json

```diff
@@ -29,7 +30,9 @@
 	},
 	"devDependencies": {
 		"@directus/tsconfig": "workspace:*",
+		"@types/node": "20.2.1",
```

Comment:
Let's use the same version as in the other packages: https://github.com/directus/directus/blob/3409821e3aae2ebc7e409ea46bbba7accf4f46c7/api/package.json#L193